js/gateio.js

```diff
@@ -656,15 +665,24 @@ module.exports = class gateio extends Exchange {
                         // Fee is in %, so divide by 100
                         'taker': this.parseNumber (Precise.stringDiv (takerPercent, '100')),
                         'maker': this.parseNumber (Precise.stringDiv (makerPercent, '100')),
-                        'contractSize': this.safeString (market, 'contractSize', '1'),
+                        'contractSize': this.safeString (market, 'quanto_multiplier'),
+                        'precision': {
+                            'amount': this.parseNumber ('1'),
+                            'price': pricePrecision,
+                        },
                         'limits': {
                             'leverage': {
+                                'min': this.safeNumber (market, 'leverage_min'),
```

Comment:
I had taken this property out due to it being equal to 1 on every single market, and because not every exchange has this property. I also thought it could possibly cause confusion, because lots of margin markets have a static leverage set (like on binance when cross is 3x and isolated 5x, 10x, ...)